src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,81 @@
+package duke;
+
+import java.util.ArrayList;
+
+import duke.tasks.Task;
+
+public class TaskList {
+    private final ArrayList<Task> taskList;
+
+    /**
+     * Initializes TaskList to a new list of tasks
+     */
+    public TaskList() {
+        this.taskList = new ArrayList<>();
+    }
+
+    /**
+     * Initializes TaskList to the specified list of tasks
+     * 
+     * @param taskList list of tasks
+     */
+    public TaskList (ArrayList<Task> taskList) {
+        this.taskList = taskList;
+    }
+
+    /**
+     * Adds the specified task to the list
+     * @param t task to be added
+     */
+    public void addTask(Task t) {
+        taskList.add(t);
+    }
+
+    /**
+     * Removes a task from the list
+     * @param index position of task in the list to be removed
+     */
+    public void removeTask (int index) {
```

Comment:
I think there should not be a space here between the method name and parameters according to the coding standard :)